<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATPMainWindow</class>
 <widget class="QMainWindow" name="ATPMainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>25</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>25</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ATPrase</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="startMenu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionloadADP"/>
    <addaction name="actionupdateATP"/>
    <addaction name="actionrunATP"/>
   </widget>
   <widget class="QMenu" name="settingMenu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionnetWork"/>
   </widget>
   <addaction name="startMenu"/>
   <addaction name="settingMenu"/>
  </widget>
  <action name="actionloadATP">
   <property name="text">
    <string>加载ATP</string>
   </property>
  </action>
  <action name="actionupdateATP">
   <property name="text">
    <string>更新ATP</string>
   </property>
  </action>
  <action name="actionloadADP">
   <property name="text">
    <string>加载ADP</string>
   </property>
  </action>
  <action name="actionrunATP">
   <property name="text">
    <string>运行项目</string>
   </property>
  </action>
  <action name="actionsetScript">
   <property name="text">
    <string>设置脚本</string>
   </property>
  </action>
  <action name="actiondrawATP">
   <property name="text">
    <string>设计电路图</string>
   </property>
  </action>
  <action name="actionnetWork">
   <property name="text">
    <string>网络设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
